app/controllers/business_entities_controller.rb

```diff
@@ -27,16 +23,28 @@ def edit
 
   def update
     authorize business_entity_service.build_update(business_entity_params)
+    persist_and_redirect(:edit)
+  end
+
+  def confirm
```

Comment:
At first sight, this confused me as to what is it confirming? I'm not insisting, but wouldn't it make sense to be `confirm_deactivate` to be clear what the action does?